### ISSUE TITLE
Totalsegmentator_mri fails for non-whole-body-MRI and contrast enhancement

### ISSUE THREAD

**Issue by Nanex101195** [wasserth/TotalSegmentator#316]
I have abdominal T1-VIBE sequences and totalsegmentator always predicts some lung, where never has been any lung visible. Also segmentation of other organs is not really competitive. I tried it with native T1, as well as contrast enhanced T1s. With contrast enhancement of the liver for example, the model fails completely to predict anything.

How can you claim a "sequence independentness" in your arxiv preprint, where segmentation on MRI is not even scanner invariant in most cases?

For CT this might hold up with the absolute HU, but for MRI? In my tests, it didn't work as explained in your paper, to be honest. I think the inhomogeneities of each MRI Scanner, let alone the sequence specific intensity distributions are not really possible to overcome. Can you explain to me, how you were satisfied with the "sequence independent" claim? I can't understand this in any way and want to understand your thought process about this.

**Comment by wasserth** [comment]
Thanks for the feedback. For all the MR images we tried it worked well. Maybe there is some other problem. What is the command you used? Would it be possible to share the dataset so I can try to reproduce the issue?

**Comment by Yes25** [comment]
I might have a similar problem. While the segmentations on CT-Images are impressive, I can't reproduce the results on MRIs. I had no full-body-MRIs available and therefore tested on this dataset: [http://medicaldecathlon.com/] (Heart, prostate and brain). For example, on the heart MRIs there are other organs like lung, liver and vertebra but the segmentations I got don't seem to catch any of these structures. 

I tested via a python script and called totalSegmentator like this:

`totalsegmentator(img, out_dir, task="total_mr", verbose=False, statistics=True, body_seg=True, force_split=False, preview=True)`

I also tried to rotate the images as mentioned in the README.md, but the results showed no improvements. 
Am I using the totalsegmentator wrong or does it not perform as good on the used data?
Anyway, thanks for the great work!
